### ISSUE TITLE
Can't use timestamps plugin from inside a repo

### ISSUE THREAD

**Issue by alexandru-calinoiu** [rom-rb/rom-sql#189]
My repo look like 

```
    class Entries < IngestService::Repository[:entries]
      relations :ingest_steps
      commands :create, use: :timestamps
   end
```

I expect that calling `Entries.new.create(args)` will add timestamps to my entry, but that is not the case.

The `Timestamp` plugin get's applied at runtime when I call create, only then `ClassInterface` is extended adding `timestamps` method that can be used to configure the colmuns. Without calling `timestamps` the `InstanceMethods` will never be included and thus timestamps never added.

Unfortunately my understanding of the plugins is very limited, they look like being targeted mostly to commands, will love some pointers for the right direction.

**Comment by alexandru-calinoiu** [comment]
One easy way to solve it is for the timestamp_column to have default values, but is not complete as one won't be able to configure them from a repo.

**Comment by solnic** [comment]
Ah, right. I'm sorry I forgot we didn't make `timestamps` plugin work with repos yet. This will be addressed in rom 4.0 as we need to improve plugin architecture first.

**Comment by solnic** [comment]
This issue was moved to rom-rb/rom#467
